R/fmri_prereg.R

```diff
@@ -0,0 +1,50 @@
+#' fmri preregistration template
+#'
+#' Knit a PDF document using fmri related questions (based on Jessica Flannery, see: https://osf.io/dvb2e/)
```

Comment:
Could you adjust this a little, maybe based on the corresponding `template.yml`? E.g. 

> Knit a PDF document using the fMRI preregistration template suggested by Jessica Flannery (https://osf.io/dvb2e/)